javascript-functions/main.js

```diff
@@ -0,0 +1,43 @@
+function convertMinutesToSeconds(minutes, seconds) {
+  var multiply = minutes * seconds;
+  return multiply;
+}
+
+var convertMinutesToSecondsResult =
+convertMinutesToSeconds(5, 60);
+console.log('convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function name(Hey, Beavis) {
+  var name = 'Hey, Beavis';
+  return name;
+}
+
+var nameResult = name('Hey, Beavis');
+console.log('nameResult:', nameResult);
```

Comment:
This function needs to be invoked with `name('Beavis')` and should return `Hey, Beavis`. Also, if we invoke it with `name('Ryan')` then it should return `Hey, Ryan`.